assets/scripts/lobby/options.js

```diff
@@ -738,6 +738,41 @@ $(document).ready(function() {
     }
 });
 
+function unhighlightAllPlayers(){
+
+    var usernames=Object.keys(selectedChat)
+
+    usernames.forEach(user => {
+        selectedChat[user] = false;
+        var chatItems = $(".room-chat-list li span[username='" + user + "']");
+        chatItems.css("background-color", "transparent");
+    });
+}
+
+function setAllHighlightColorsToYellow(){
+
+    var color="#ffff9e"
+
+    for(var i = 0; i < 10; i++){
+        docCookies.setItem('player' + i + "HighlightColour", color, Infinity);
```

Comment:
Why not just put in an exception in the highlight area (addToRoomChat() under assets/scripts/lobby/chat4.js) to set it to yellow if the option is ticked or not, instead of flat out overwriting the default colour values? There should be a way to retain the original colours. If someone missclicks it and loses all their data getting it back would be a pain. Especially if later on we're going to introduce username specific colours for people to assign locally.